tests/GrabyTest.php

```diff
@@ -897,26 +862,6 @@ public function testContentLinksRemove(): void
         $this->assertFalse($res['native_ad']);
     }
 
-    public function testMimeActionNotDefined(): void
```

Comment:
Why was this removed? Should be probably mentioned in the commit message.

Reply:
I removed it because it's now checked by the OptionResolver, so the behavior changed (ie: it's not more a silent error): https://github.com/j0k3r/graby/blob/0c7cd9375225a9476365384990e3deabf2f70a1f/src/GrabyConfig.php#L74

Reply:
Oh, right. I guess that can have `array<string, array{name: string, action: 'link'|'exclude'}>` annotation: https://phpstan.org/writing-php-code/phpdoc-types